src/olympia/migrations/1012-drop-eventlog-table.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS `eventlog`;
```

Comment:
Just making sure, this was the table that has not been written to since 2012 or so? In that case I doubt ops needs special notice.

Reply:
correct.  It was used by a previous iteration of the reviewer tools but everything has been in ActvityLog since then.